packages/react-core/src/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -123,6 +137,34 @@ import { MenuToggle } from '@patternfly/react-core';
 </React.Fragment>
 ```
 
+
+### With icon/image and text
+
+```ts
+import React from 'react';
+import { MenuToggle, Avatar } from '@patternfly/react-core';
+import CogIcon from '@patternfly/react-icons/dist/esm/icons/cog-icon';
+
+<React.Fragment>
+  <MenuToggle icon={<CogIcon />} variant="secondary">Icon</MenuToggle>{' '}
+  <MenuToggle icon={<CogIcon />} variant="secondary" isDisabled>Icon</MenuToggle>
+</React.Fragment>
+```
+
+### With avatar and text
+
+```ts
+import React from 'react';
+import { MenuToggle, Avatar } from '@patternfly/react-core';
+import avatarImg from './avatarImg.svg';
```

Comment:
Could you import from the avatar component like the dashboard wrapper does? Then we're using the same image consistently. https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/demos/examples/DashboardHeader.js#L30